tests/helpers/module-for-acceptance.js

```diff
@@ -0,0 +1,23 @@
+import { module } from 'qunit';
+import startApp from '../helpers/start-app';
+import destroyApp from '../helpers/destroy-app';
+
+export default function(name, options = {}) {
```

Comment:
'export' is only available in ES6 (use esnext option).<br>'default parameters' is only available in ES6 (use esnext option).